decisions/README.md

```diff
@@ -0,0 +1,7 @@
+# Decisions documentation
+
+Greatly inspired by ADRs, this directory contains important decisions about the `simulator` code.
+
+# Useful documentation
+
+- https://github.com/joelparkerhenderson/architecture-decision-record
```

Comment:
@AgustinRamiroDiaz, please, let's add the index of all the ADRs here.

Reply:
Added!